autosaloon/setup_and_security.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+DJANGO_KEY = os.getenv('DJANGO_KEY')
+
+POSTGRES_DB = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'postgres',
+        'USER': os.getenv('DB_USER'),
+        'PASSWORD': os.getenv('DB_PASS'),
+        'HOST': os.getenv('DB_HOST'),
+        'PORT': os.getenv('DB_PORT'),
+        }
+    }
+
```

Comment:
For now, you don't need to move it into another module, keep in in settings file

Reply:
Done. Also added to .env database NAME as I have different db names on my PC and laptop.